src/vehicles/european.vehicle.ts

```diff
@@ -165,6 +165,14 @@ export default class EuropeanVehicle extends Vehicle {
 
       const fullStatus = cachedResponse.body.resMsg.vehicleStatusInfo;
 
+      const ccs2Response = this.updateRates(
```

Comment:
This means we'd do one request for the old endpoint and another on for CCS2, do we want that?

I'm pretty out of the loop regarding the new API.